### PR TITLE
Line wrapping adjustments

### PR DESCRIPTION
**Pull request by MCelliotG**

### CODE REVIEW

**Comment by MCelliotG** [comment]
Strings changed in the .ino file need to support wrapping as well.

File: TEF6686_ESP32.ino [hunk 1]
```diff
@@ -558,8 +558,8 @@ void setup() {
     EEPROM.writeByte(EE_BYTE_ROTARYMODE, rotarymode);
     EEPROM.commit();
     analogWrite(CONTRASTPIN, ContrastSet * 2 + 27);
-    tftPrint(0, myLanguage[language][1], 155, 85, ActiveColor, ActiveColorSmooth, 28);
-    tftPrint(0, myLanguage[language][2], 155, 115, ActiveColor, ActiveColorSmooth, 28);
+    tftPrint(0, myLanguage[language][1], 155, 70, ActiveColor, ActiveColorSmooth, 28);
+    tftPrint(0, myLanguage[language][2], 155, 130, ActiveColor, ActiveColorSmooth, 28);
     while (digitalRead(BWBUTTON) == LOW) delay(50);
```
Comment (MCelliotG) [inline TEF6686_ESP32.ino:563]: These strings appear in the splash screen when using button combinations. They need to support wrapping as well.

**Comment by PE5PVB** [comment]
I don't think text wrapping will work there. I'll add it soon.

**Comment by MCelliotG** [comment]
It's worth a try!!!
BTW, with the latest commit PI does not appear, can you check it out?

**Pull request merged**